label_studio/tasks/models.py

```diff
@@ -992,6 +992,50 @@ class Meta:
         db_table = 'prediction'
 
 
+class FailedPrediction(models.Model):
+    """
+    Class for storing failed prediction(s) for a task
+    """
+    message = models.TextField(
+        _('message'),
+        default='',
+        blank=True,
+        null=True,
+        help_text='The message explaining why generating this prediction failed'
+    )
+    error_type = models.CharField(
+        _('error_type'),
+        max_length=512,
+        default=None,
+        null=True,
+        help_text='The type of error that caused prediction to fail'
+    )
+    ml_backend_model = models.ForeignKey(
```

Comment:
Ah... is this what we do for regular Predictions? Unfortunate that we have different attributes for Prompts vs ML Backend, we'll have to fix that eventually.
I figure blank=True here though, since it would be blank for Prompts-based failed preds?

Reply:
blank is not an option for FKs, null is